rollup-cli/tools/resourcesBot/sendBot.js

```diff
@@ -0,0 +1,117 @@
+const axios = require("axios");
```

Comment:
this file should not be in resources
Should be in `tools` folder

Reply:
Is in helpers now